doc/api_conf.rst

```diff
@@ -44,6 +49,18 @@ Configurable settings of the :class:`dnf.Base` object are stored into a :class:`
 
     Debug messages output level, in the range 0 to 10. Default is 2.
 
+
+  .. attribute::defaultyes
+
+    Boolean option, if set to ``True`` on any user input asking for confirmation
+    (e.g. after transaction summary) the default answer is ``yes``. The default is ``False``.
+
+  .. attribute:: defaultno
+
+    Boolean option, if set to ``True`` on any user input asking for confirmation
+    (e.g. after transaction summary) the default answer is ``no``. This option will override ``defaultyes`` and should be used with caution. Default is ``False``.
+  
+
```

Comment:
If `assumeno`, `defaultyes` and `defaultno` are inapropiate for the Python API I will revert the changes to `api_conf`